index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="header_logo">My Logo</div>
+        <div class="header_menu">
+          <img class="menu_icon" src="/images/menu-icon.svg" alt="menu" />
```

Comment:
As you can see from the screenshot below your icon image is not displaying. I think your path is not right. Please check your image path. I can only see the text menu rather than the hamburger icon

### Your design
![menu](https://user-images.githubusercontent.com/26736582/198091189-4444707c-64ab-40a7-b3ca-e64bd2c0307c.png)

### Figma design

![figma1](https://user-images.githubusercontent.com/26736582/198092676-4736a8db-377f-4767-b333-5de8d9a894f3.png)